ql/src/java/org/apache/hadoop/hive/ql/parse/StorageFormat.java

```diff
@@ -93,6 +106,14 @@ public boolean fillStorageFormat(ASTNode child) throws SemanticException {
     return true;
   }
 
+  private String processStorageHandler(String name) throws SemanticException {
+    if (StorageHandlerTypes.ICEBERG.name().equalsIgnoreCase(name)) {
```

Comment:
Shall we make this generic so it works for future StorageHandlerTypes types? i.e. iterate through the enum values and if `name` matches any enum value, then reset the `name` with `enum.name()` accordingly.

Reply:
I was thinking about HBase / Kudu / Druid, but that probably merits another PR 😄

Reply:
I think the code change to make it generic would be very small, probably a 4-line for loop, so we might as well add it into this PR, but I don't feel strongly, just a suggestion.

Reply:
Done.